### ISSUE TITLE
pstops does not recognise left from right

### ISSUE THREAD

**Issue by opohorel** [rrthomas/psutils#36]
Hi, I've got an report saying, that pstops doesn't recognise left from right.

```
These two commands :

$ pstops -pa4 -S '0L(1w,0)+0L(1w,.5h)' <file.ps

$ pstops -pa4 -S '0L(1w,0)+0R(1w,.5h)' <file.ps

produce the same output, yet the top half should be rotated to the left in the first case, to the right in the second. Also, in the second case, the offset (1w,.5h) should be (0,1h) to keep the result on the same page.
```

**Comment by rrthomas** [comment]
This is not a valid command with the current version (2.05) of pstops (which has no `-S` flag). Closing.

**Comment by rrthomas** [comment]
Thanks for this report; the problem was in normalizing the angle: one step of this process was both unnecessary *and* done incorrectly.

**Comment by rrthomas** [comment]
I have released version 2.06 with the fix.
